SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h

```diff
@@ -99,7 +99,7 @@ FORCEINLINE SubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor, const F
 
 	for (auto& SubobjectInfoPair : Info.SubobjectInfo)
 	{
-		UObject* Subobject = Actor->GetDefaultSubobjectByName(SubobjectInfoPair.Value->SubobjectName);
+		UObject* Subobject = StaticFindObjectFast(UObject::StaticClass(), Actor, SubobjectInfoPair.Value->SubobjectName);
```

Comment:
`GetDefaultSubobjectByName()` already calls `StaticFindObjectFast()` underneath, except when it's garbage collecting. Was it the `!Object->IsDefaultSubobject()` line that was breaking it?

We are now losing the `else()` codepath we previously had within `GetDefaultSubobjectByName()`. Is this ok?

Reply:
Yeah, it was failing `IsDefaultSubobject`, and so returning null where a subobject did indeed exist. I had a look with @improbable-valentyn, and we're fairly confident that the else statement would not be reached from this call (will always happen on the game thread, so won't be called whilst in the middle of saving or garbage collecting, whereas `GetDefaultSubobjectByName` could get called from other places so requires the added security)